src/data/lumine-table.tbl

```diff
@@ -0,0 +1,117 @@
+00=\x00
+01=\x01
+17=\x17
+1C=\x1C
+26=\x26
+30=\x30
+3E=\x3E
+44=\x44
+4A=\x4A
+4C=\x4C
+56=\x56
+
+FF=\end
+FE=\n
+FD=\blank
+00= 
+01=!
+02="
+03=#
+04=$
+05=%
+06=&
+07='
+08=(
+09=)
+0A=*
+0B=+
+0C=,
+0D=-
+0E=.
+0F=/
+10=0
+11=1
+12=2
+13=3
+14=4
+15=5
+16=6
+17=7
+18=8
+19=9
+1A=:
+1B=;
+1C=<
+1D==
+1E=>
+1F=?
+20=@
+21=A
+22=B
+23=C
+24=D
+25=E
+26=F
+27=G
+28=H
+29=I
+2A=J
+2B=K
+2C=L
+2D=M
+2E=N
+2F=O
+30=P
+31=Q
+32=R
+33=S
+34=T
+35=U
+36=V
+37=W
+38=X
+39=Y
+3A=Z
+3B=[8B]
+3C=[8C]
+3D=[8D]
+3E=[8E]
+3F=[8F]
+40=`
+41=a
+42=b
+43=c
+44=d
+45=e
+46=f
+47=g
+48=h
+49=i
+4A=j
+4B=k
+4C=l
+4D=m
+4E=n
+4F=o
+50=p
+51=q
+52=r
+53=s
+54=t
+55=u
+56=v
+57=w
+58=x
+59=y
+5A=z
+5B={
+5C=|
+5D=}
+5E=~
+5F=[AF]
+
+F9=\cness
+FA=\cpaula
+FB=\cjeff
+FC=\cpoo
+/00
```

Comment:
This part changes the byte used for "end" of a string defined with `.str`. You can just set it as /FF and not need to use the \end manually in the actual text.

Reply:
Cool!